roles/kubernetes-apps/ingress_controller/cert_manager/templates/clusterrole-cert-manager.yml.j2

```diff
@@ -4,7 +4,7 @@ kind: ClusterRole
 metadata:
   name: cert-manager
   labels:
-    app: cert-manager
+    k8s-app: cert-manager
     chart: cert-manager-v0.4.1
     release: cert-manager
     heritage: Tiller
```

Comment:
why is there 'tiller' label when it's not deployed via tiller ?

Reply:
Same note for chart and release.

Reply:
Copy & Past from official manifest example for simplify integration overhead (i.e. lesser result during diff):

-   https://github.com/jetstack/cert-manager/blob/master/contrib/manifests/cert-manager/with-rbac.yaml#L95